content/concepts/asset-pricing.md

```diff
@@ -42,6 +42,28 @@ The image below shows how to set the Dynamic pricing of an asset in the Ocean's
 
 Ocean Protocol also allows publishers to set the pricing using ocean.js and ocean.py library.
 
+### Asset price
+
+#### Action: Add liquidity from pool
```

Comment:
"Add liquidity from pool' doesn't sound correct. Is adding to the pool. Let's keep it simple: "Add liquidity" cause we explain the things in the next paragraph. The same for "Remove liquidity from pool"